### DISCUSSION TITLE
When upload an mp3 file is added a ".txt" extension

### DISCUSSION THREAD

**Discussion by rgaricano** [open-webui/open-webui#5634] [General]
# Bug Report
When upload an mp3 file through upload chat box it is upload to docs but with a plus ".txt" extension.

## Installation Method
git clone

## Environment

- **Open WebUI Version: 0.3.21
- **Ollama (if applicable): 0.3.11
- Ubuntu 24.04
- Chromium Versión 128.0.6613.13

## Expected Behavior:

Upload mp3 file to doc folder
eg: mi_explosion_03_hpx.mp3

## Actual Behavior:

Upload mp3 file to doc folder but add a ".txt" extension
eg: mi_explosion_03_hpx.mp3.txt

**Comment by tjbck** [comment]
audio files are automatically transcribed then uploaded as a text file.

**Reply by rgaricano** [reply]
ok, 
but it's not a good idea save the transcription only,  maybe it is bad for the model used in that moment, or if later we want to use it as audio or with other model more accurate.....
